solr-synonyms-api/devops/gcp/clouddeploy.yaml

```diff
@@ -45,6 +45,19 @@ serialPipeline:
       container-name: "namex-solr-synonyms-api-test"
       service-account: "sa-api@a083gt-test.iam.gserviceaccount.com"
       cloudsql-instances: "a083gt-test:northamerica-northeast1:namex-db-test"
+ - targetId: a083gt-integration
+   profiles: [sandbox]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "sandbox"
```

Comment:
this needs to be "production",  CD pipeline maps deploy-env to DEPLOYMENT_ENV, which is mapped in the code to actual configuration object, e..g in the code I see these values:

CONFIGURATION = {
    'development': 'config.DevConfig',
    'testing': 'config.TestConfig',
    'production': 'config.Config',
    'default': 'config.Config'
}   https://github.com/bcgov/namex/blob/011ec080705f56f685487567c5d24f830b50352c/solr-synonyms-api/config.py#L10 --- they are mapped here: https://github.com/bcgov/namex/blob/011ec080705f56f685487567c5d24f830b50352c/solr-synonyms-api/synonyms/__init__.py#L24